src/acsets/schemas/catlab/SummationDecapode.json

```diff
@@ -0,0 +1,213 @@
+
+{
+    "version": {
+        "ACSets": "0.0.0",
+        "ACSetSchema": "0.0.1"
+    },
+    "Ob": [
+        {
+            "name": "Var"
```

Comment:
Ideally, all `Ob`s have a "title" that has a human readable name. The schema should provide some kind of information for anyone who is getting data in this shape what the data is